env/backend.env

```diff
@@ -33,3 +33,7 @@ UWSGI_THREADS=35
 
 TIKA_SERVER_ENDPOINT=http://tika:9998/
 TIKA_CLIENT_ONLY=True
+
+# _JAVA_OPTIONS for Opensearch container are not consistent between different CPU architectures e.g ARM and x86.
+# Here, anyone can set options like -XX:UseSVE=0 based on their CPU.
+JAVA_OPTIONS=
```

Comment:
Since it's null by default, could move this to `backend.local.example.env` instead, and anyone who needs it can add  `JAVA_OPTIONS=-XX:UseSVE=0` to their `backend.local.env` file.

Reply:
Looks good